core/src/main/java/uk/ac/bbsrc/tgac/miso/core/service/printing/Driver.java

```diff
@@ -90,7 +90,7 @@ public String encode(Barcodable barcodable, int copies) {
         sb.append("\n");
       };
       offset.addAndGet(multiline(28, 5, barcodable.getLabelText(), writer));
-      multiline(28, 5 - offset.get(), StringEscapeUtils.unescapeHtml(barcodable.getBarcodeExtraInfo()), writer);
+      multiline(28, 5 - offset.get(), unescapeHtml(barcodable.getBarcodeExtraInfo()), writer);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') "NullPointerException" will be thrown when invoking method "multiline()". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2259)